javascript/Round.js

```diff
@@ -37,6 +37,6 @@ createRound() {
     return categories.map((category) => {
       return this.data.categories[category]
     })
-  }
+}
```

Comment:
Looks like this `createCategories` method already exists up above? Just make sure you don't have duplicate code -- it's hard to tell just by looking at the PR.